### ISSUE TITLE
Missing alt text for grouped nav nodes arrow

### ISSUE THREAD

**Issue by JohannesDoberer** [SAP/luigi#3007]
![Bildschirm­foto 2022-11-24 um 21.15.11.png](https://images.zenhubusercontent.com/5c346c4d86008d0a2e182a83/969c923a-6076-4c5c-994e-b7ad7fde75c4)

Text should be configurable.
